### ISSUE TITLE
The bar is not displayed correctly

### ISSUE THREAD

**Issue by Fan73** [rianadon/timer-bar-card#33]
**Describe the bug**

Hello,
I use "Timer Bar Card" for many weeks already
and everything was ok.
Today on the Home Assistant system I have some 
updates and suddenly there were no
bars were no longer visible.
I have added some values and now have the following result:

![image](https://user-images.githubusercontent.com/61880929/146648205-c884a139-430a-4311-a2aa-91c231598263.png)

When the bar is displayed, there are suddenly 2-3 lines.
Previously, everything was displayed in one line.

![image](https://user-images.githubusercontent.com/61880929/146648432-3d34c665-911d-448a-bc35-d8ece6a094a1.png)

what can i do now?

Greetings...

**Your card configuration**
```yaml
type: custom:timer-bar-card
entity: timer.treppenlichtaus
name: Treppe
attribute: duration
unit: sec
bar_foreground: '#eee'
bar_background: 'var(--mdc-theme-primary, #6200ee)'
bar_height: 8px
bar_width: 80%
bar_direction: rtl
text_width: 300px
```

**Debug information**
Please add `debug: true` to your configuration and either take a screenshot or paste the output here!

**Comment by rianadon** [comment]
Thanks for reporting the issue! This is fixed in v1.15!

**Comment by Fan73** [comment]
Hello,
thank you very much for your wonderful work.
Thank you very much,
it works perfectly...

Greetings from Halle
